### PR TITLE
Add more info if custom ns is used to install Istio

### PR DESCRIPTION
**Pull request by shamsher31**
Attempt to fix https://github.com/istio/istio/issues/30897

### CODE REVIEW

**Review by smawson — APPROVED** [review]

**Comment by shamsher31** [comment]
@linsun @nrjpoddar PTAL.

**Comment by shamsher31** [comment]
@linsun @nrjpoddar ping.

File: operator/v1alpha1/operator.proto [hunk 1]
```diff
@@ -86,7 +86,8 @@ message IstioOperatorSpec {
     // Never implemented; replaced by revision.
     string resource_suffix = 14 [deprecated = true];
     // Namespace to install control plane resources into. If unset, Istio will be installed into the same namespace
-    // as the `IstioOperator` CR.
+    // as the `IstioOperator` CR. You must also set `values.global.istioNamespace` if you wish to install Istio in
+    // a custom namespace. If you enabled CNI, you also need to set `values.cni.excludeNamespaces[0]`.
```
Comment (nrjpoddar) [inline operator/v1alpha1/operator.proto:90]:
```suggestion
    // a custom namespace.
    // If you have enabled CNI, you must  exclude this namespace by adding it to the list `values.cni.excludeNamespaces[0]`.
```

Comment (shamsher31) [reply operator/v1alpha1/operator.proto]: Thanks.

**Comment by shamsher31** [comment]
cc: @linsun

**Review by nrjpoddar — APPROVED** [review]

**Pull request merged**